app/views/communities/_bannedusers.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="card" style="width: 40rem;">
+  <div class="card-header">Banned Users <i class="fa fa-info-circle text-primary ml-1"></i></div>
+  <% if @banneduser.empty? %>
+    <p>***No Banned Users***</p>
+  <% else %>
+    <% @banneduser.each do |user| %>
```

Comment:
can you change the variable to follow conventions? it has to be snake case when there are more than 1 words

Reply:
Resolved by creating new Branch #113